engine/server/application/LoginServer/src/shared/LoginServer.cpp

```diff
@@ -480,7 +480,9 @@ void LoginServer::receiveMessage(const MessageDispatch::Emitter &source, const M
                         break;
                     }
                 }
-                DEBUG_REPORT_LOG(!found, ("Tried to remove a connection server that wasn't in our list.\n"));
+                if(!found) {
+                    DEBUG_REPORT_LOG(true, ("Tried to remove a connection server that wasn't in our list.\n"));
+                }
                 m_clusterStatusChanged = true;
```

Comment:
Because the DEBUG_REPORT_LOG() doesn't "use" the `found` var so the compiler thinks it's a wasted value.  Doing this "uses" the `found` var and therefore shuts up the compiler.